include/awkward/Content.h

```diff
@@ -63,6 +64,7 @@ namespace awkward {
     virtual bool mergeable(const std::shared_ptr<Content>& other, bool mergebool) const = 0;
     virtual const std::shared_ptr<Content> merge(const std::shared_ptr<Content>& other) const = 0;
     virtual const std::shared_ptr<SliceItem> asslice() const = 0;
+    virtual const std::shared_ptr<Content> rpad(int64_t length, int64_t axis) const = 0;
```

Comment:
@jpivarski - I think, I'll change `length` to `rpad_width` everywhere. There is a `length()` method already that may confuse the compiler.

Reply:
I changed my mind, it should be `length`